lib/browser.js

```diff
@@ -172,10 +177,14 @@ class Browser {
   }
 
   disconnect (reason) {
-    this.log.warn(`Disconnected (${this.disconnectsCount} times)${reason || ''}`)
-    this.setState(DISCONNECTED)
+    this.log.warn(`Disconnected (${this.disconnectsCount} times) ${reason || ''}`)
```

Comment:
Nit: revert this line so as to keep a better blame layer.

Reply:
This was a drive by fix, so let's leave it.